docker-for-windows/index.md

```diff
@@ -410,6 +410,8 @@ See also [Verify domain user has permissions for shared
 drives](troubleshoot.md#verify-domain-user-has-permissions-for-shared-drives-volumes)
 in Troubleshooting.
 
+Note that you can share local drives with your _containers_ but not with Docker Machine created machines. See [Can I share local drives and filesystem with my Docker Machine VM's?](can-i-share-local-drives-and-filesystem-with-my-docker-machine-vms?) in the FAQ's.
```

Comment:
Maybe instead of "Note that", just make it a note?

"Docker Machine created machines" maybe "nodes created by Docker machine"?